modules/deploy/pages/deployment-option/cloud/cloud-overview.adoc

```diff
@@ -1,113 +1,91 @@
-** A multi-zone cluster is deployed across three availability zones. This
-provides higher resiliency in the event of a failure in one of the zones.
-* Private networking using VPC peering
-* Ability to export metrics to a 3rd-party monitoring system
-* Managed connectors
-* Unlimited retention with glossterm:Tiered Storage[]
+. You bootstrap a virtual machine (VM) in your VPC. This VM spins up the agent
+and the required infrastructure.
+. The agent communicates with the control plane to pull the cluster specifications.
 
-=== Dedicated Cloud
+Redpanda assigns the necessary IAM policies required to run the agent at the time of deployment.
 
-With Dedicated Cloud, you host your data in Redpanda's VPC, and Redpanda handles provisioning, operations, and maintenance. Dedicated clusters are single-tenant deployments launched from the Redpanda
-control plane. The Redpanda cluster is created in Redpanda's VPC, and the
-endpoints are exposed so you can produce or consume events.
+Available on AWS or GCP. 
 
-=== BYOC
+==== Sign up for BYOC
 
-With BYOC, you deploy Redpanda in your own VPC, so all data is
-contained in your own environment. There are no inbound connections from the
-control plane to the data plane. This provides an additional layer of security
-and isolation. Redpanda handles provisioning, monitoring, and upgrades. Setting up a BYOC cluster is initiated from the Redpanda control plane. This is
-a two-step process performed using `rpk`:
+To start using BYOC, contact https://redpanda.com/try-redpanda?section=enterprise-trial[Redpanda sales^] to request a private offer, and you can be billed either directly or through Google Cloud Marketplace or AWS Marketplace. 
 
-. You bootstrap a virtual machine (VM) in your VPC. This VM spins up the agent
-and the required infrastructure.
-. The agent communicates with the control plane to pull the cluster specifications.
+== Serverless vs Dedicated/BYOC
+
+Serverless clusters are a good fit for the following:
+
+* Starter and growing workloads
+* Spiky workloads (that is, development environments, systems that only occasionally get busy, or workloads that come and go)
+* Fast and dynamic cluster creation: you can use a Serverless cluster as an isolated container for topics
+
+With Serverless, you only pay for what you consume, without any commitment. A cluster is created instantly, so you can surface it in your applications (for example, for tenant isolation). If your workload increases, you can migrate it to a Dedicated or BYOC cluster.
+
+Dedicated and BYOC clusters offer the following features:
 
-When provisioning BYOC, at the time an agent is deployed, Redpanda assigns the
-necessary IAM policies required to run the agent.
+* Single-zone or multi-zone availability (a multi-zone cluster provides higher resiliency in the event of a failure in one of the zones)
+* Private networking using VPC peering
+* Ability to export metrics to a 3rd-party monitoring system
+* Managed connectors
```

Comment:
I believe we aim to remove this limitation someday, so, something to be aware of in the future